docs/authentication.adoc

```diff
@@ -0,0 +1,30 @@
+== Authentication
+
+There are two authentication-methods within the API. The following chapter shall give an overview on how to create and then use the generated tokens for making requests to the API. +
+
+Please take a look at the Swagger-Documentation of the API to see which tokens needs to be provided for using which endpoint or how the Body-Payload for the different endpoints need to look like.
+
+=== Tokens
+
+The API uses two kinds of tokens. 
+
+* **API-Token:** 
+** The API-Token is a static token, which gets set in the environment-variables as `APPROVED_TOKENS`. 
+** This token is used to authenticate an 'admin-App' against the API. The API-Token is used to generate **Bearer-Tokens**. 
+** For generating more than one token they can be passed to the environment-variables as a comma-seperated list.
+* **Bearer-Token**
+** The Bearer-Token contains the **keyspace**, **Client-ID** and **Scopes**
+** Currenty there are two Scopes available `Data.Read` and `Data.Write`
+** Client-ID is a unique UUID, which gets generated when generating a new token
+** The token isn't valid forever, it will expire after some time
+** The **Keyspace** is the ServerID of the Discord-Bot which consumes the API
+** **!IMPORTANT** currently Datastax Astra isn't able to generate a new Keyspace on the fly. Therefor the keyspace needs to be created manually before being able to use a new one.
```

Comment:
the `important` section might stand out more using the `Admonitions` from [Asciidoctor](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#admonitions)

It will look like this ...

<img width="1070" alt="Screenshot 2021-08-07 at 08 11 50" src="https://user-images.githubusercontent.com/624760/128591985-558ece8b-6626-40bb-b732-11996cd126d1.png">